android/capacitor/src/main/java/com/getcapacitor/plugin/notification/LocalNotificationManager.java

```diff
@@ -174,6 +175,12 @@ private void buildNotification(NotificationManagerCompat notificationManager, Lo
     mBuilder.setOnlyAlertOnce(true);
 
     mBuilder.setSmallIcon(localNotification.getSmallIcon(context));
+
+    String iconColor = localNotification.getIconColor();
+    if (iconColor != null) {
+      mBuilder.setColor(Color.parseColor(iconColor));
```

Comment:
this crash if the color is not valid, put it inside a try catch (check other plugins like Browser or Statusbar)